pvlib/tests/test_tracking.py

```diff
@@ -486,3 +488,79 @@ def test_LocalizedSingleAxisTracker___repr__():
+    # calculate tracker axis zenith
+    axis_tilt = tracking.calc_tracker_axis_tilt(
+        *system_plane, axis_azimuth=axis_azimuth)
+    assert np.isclose(axis_tilt, expected_axis_tilt)
+    # calculate side slope and relative rotation
+    side_slope = tracking.calc_system_tracker_side_slope(
+        axis_azimuth, axis_tilt, *system_plane)
+    assert np.isclose(side_slope, expected_side_slope)
+    sat = tracking.singleaxis(
+        solpos.apparent_zenith, solpos.azimuth, axis_tilt, axis_azimuth,
+        max_angle, backtrack=True, gcr=gcr, side_slope=side_slope)
+    np.testing.assert_allclose(
+        sat['tracker_theta'], expected['tracker_theta'], atol=1e-7)
+    np.testing.assert_allclose(sat['aoi'], expected['aoi'], atol=1e-7)
+    np.testing.assert_allclose(
+        sat['surface_azimuth'], expected['surface_azimuth'], atol=1e-7)
+    np.testing.assert_allclose(
+        sat['surface_tilt'], expected['surface_tilt'], atol=1e-7)
+
+
+def test_slope_aware_backtracking():
+    """
+    Test validation data set from https://www.nrel.gov/docs/fy20osti/76626.pdf
+    """
+    expected_data = np.array(
+        [('2019-01-01T08:00-0500',  2.404287, 122.79177, -84.440, -10.899),
+         ('2019-01-01T09:00-0500', 11.263058, 133.288729, -72.604, -25.747),
+         ('2019-01-01T10:00-0500', 18.733558, 145.285552, -59.861, -59.861),
+         ('2019-01-01T11:00-0500', 24.109076, 158.939435, -45.578, -45.578),
+         ('2019-01-01T12:00-0500', 26.810735, 173.931802, -28.764, -28.764),
+         ('2019-01-01T13:00-0500', 26.482495, 189.371536, -8.475, -8.475),
+         ('2019-01-01T14:00-0500', 23.170447, 204.13681, 15.120, 15.120),
+         ('2019-01-01T15:00-0500', 17.296785, 217.446538, 39.562, 39.562),
+         ('2019-01-01T16:00-0500',  9.461862, 229.102218, 61.587, 32.339),
+         ('2019-01-01T17:00-0500',  0.524817, 239.330401, 79.530, 5.490)],
+        dtype=[
+            ('Time', '<M8[h]'), ('ApparentElevation', '<f8'),
+            ('SolarAzimuth', '<f8'), ('TrueTracking', '<f8'),
+            ('Backtracking', '<f8')])
+    expected_axis_tilt = 9.666
+    expected_slope_angle = -2.576
+    system_azimuth, system_zenith = 180.0, 10.0
+    axis_azimuth = 195.0
+    axis_tilt = tracking.calc_tracker_axis_tilt(
+        system_azimuth, system_zenith, axis_azimuth)
+    assert np.isclose(axis_tilt, expected_axis_tilt, rtol=1e-3, atol=1e-3)
+    side_slope = tracking.calc_system_tracker_side_slope(
+        axis_azimuth, axis_tilt, system_azimuth, system_zenith)
+    assert np.isclose(side_slope, expected_slope_angle, rtol=1e-3, atol=1e-3)
+    sat = tracking.singleaxis(
+        90.0-expected_data['ApparentElevation'], expected_data['SolarAzimuth'],
+        axis_tilt, axis_azimuth, max_angle=90.0, backtrack=True, gcr=0.5,
+        side_slope=side_slope)
+    np.testing.assert_allclose(
+        sat['tracker_theta'], expected_data['Backtracking'],
+        rtol=1e-3, atol=1e-3)
```

Comment:
Consider adding an assert on the true-tracking values here as well, maybe?  Though the TT behavior is already tested elsewhere so maybe not.